cdap-data-fabric/src/main/java/co/cask/cdap/data2/metadata/lineage/Lineage.java

```diff
@@ -27,27 +26,15 @@
  */
 public class Lineage {
   private final Set<Relation> relations;
-  private final Set<Id.Program> programs;
```

Comment:
Right now they are not being used anywhere. Having them only increased the size of code, hence removed them. It should be easy to add them back if required later.